pkgs/development/beam-modules/hex/default.nix

```diff
@@ -8,13 +8,15 @@ let
 
   pkg = self: stdenv.mkDerivation rec {
     pname = "hex";
-    version = "0.20.1";
+    version = "0.20.5";
 
+    # How to obtain `sha256`:
+    # nix-prefetch-url --unpack https://github.com/hexpm/hex/archive/v${version}.tar.gz
```

Comment:
Frankly, that's true of every single `fetchFromGitHub` case, and I'm not sure this comment is deserved here. (Or one could also use `nix-prefetch-github`).

Reply:
I wasn't aware of `nix-prefetch-github`.
But I prefer to communicate the universal usable method of `nix-prefetch-url`.

I've started to add this comment on nixpkgs, as I still find it not easy for new peoples to find out the easy way of updating a pkg with getting the `sha256`....

Reply:
But of course, I can remove it here.
But I still would add this on pkgs where the fetchFromGitHub is not do be found directly beneath the `version`.

Reply:
```
$ nix-shell -p nix-prefetch-github --run 'nix-prefetch-github hexpm hex --rev v0.20.5'
{
    "owner": "hexpm",
    "repo": "hex",
    "rev": "3f49ed54d2858f6fdb9a06c3f7b242b3dbe1d081",
    "sha256": "1wz6n4qrmsb4kkww6lrdbs99xzwp4dyjjmr8m4drcwn3sd2k9ba6"
}
```

Reply:
Thanks a lot for removing it.
I would think of some ways to communicate this in future more easily to new contributors...